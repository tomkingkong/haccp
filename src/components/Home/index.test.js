import React from 'react';
import { shallow } from 'enzyme';

import { Home } from '.';

describe('Home Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});