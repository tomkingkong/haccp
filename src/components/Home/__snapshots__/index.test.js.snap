// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should match snapshot 1`] = `
<div>
  <nav
    className="header-nav"
  >
    <h1
      className="header-text"
      onClick={[Function]}
    >
      HACCP Planner
    </h1>
    <div>
      <button
        className="signup-login"
        onClick={[Function]}
      >
        Sign Up
      </button>
      <button
        className="signup-login"
        onClick={[Function]}
      >
        Log In
      </button>
    </div>
  </nav>
  <div
    className="welcome-header-img"
  >
    <div
      className="splash-text"
    >
      <p>
        HACCP Planner offers a step-by-step, food safety plan generator to comply with the Food Safety Modernization Act and HACCP.
      </p>
      <button
        className="get-started"
        onClick={[Function]}
      >
        GET STARTED
      </button>
    </div>
  </div>
  <section
    className="welcome-container"
  >
    <div
      className="welcome-step-container"
    >
      <div
        className="welcome-step-img welcome-img-1"
      />
      <h2
        className="welcome-step-title"
      >
        Create an account
      </h2>
      <p
        className="welcome-step-text"
      >
        Enter basic information about your facility, products, and team members.
      </p>
    </div>
    <div
      className="welcome-step-container"
    >
      <div
        className="welcome-step-img welcome-img-2"
      />
      <h2
        className="welcome-step-title"
      >
        Add products
      </h2>
      <p
        className="welcome-step-text"
      >
        Start a plan for each product by adding ingredient information.
      </p>
    </div>
    <div
      className="welcome-step-container"
    >
      <div
        className="welcome-step-img welcome-img-3"
      />
      <h2
        className="welcome-step-title"
      >
        Define your process
      </h2>
      <p
        className="welcome-step-text"
      >
        Enter information about each ingredient during its life cycle.
      </p>
    </div>
    <div
      className="welcome-step-container"
    >
      <div
        className="welcome-step-img welcome-img-4"
      />
      <h2
        className="welcome-step-title"
      >
        Identify hazards
      </h2>
      <p
        className="welcome-step-text"
      >
        Identify hazards and critical control points for different processing stages.
      </p>
    </div>
    <div
      className="welcome-step-container"
    >
      <div
        className="welcome-step-img welcome-img-5"
      />
      <h2
        className="welcome-step-title"
      >
        Review your plan
      </h2>
      <p
        className="welcome-step-text"
      >
        View and print the completed version of your HACCP plans.
      </p>
    </div>
    <div
      className="welcome-step-container"
    >
      <div
        className="welcome-step-img welcome-img-6"
      />
      <h2
        className="welcome-step-title"
      >
        Edit anytime
      </h2>
      <p
        className="welcome-step-text"
      >
        Review and make changes to your product plans anytime, anywhere.
      </p>
    </div>
  </section>
</div>
`;
